pkg/constants/constants.go

```diff
@@ -203,6 +203,9 @@ const (
 	// CRIContainerdConfig is the path to the config for the containerd instance that provides the CRI.
 	CRIContainerdConfig = "/etc/cri/containerd.toml"
 
+	// TalosDirectoryEnvVar is the environment variable for setting the Talos directory path.
+	TalosDirectoryEnvVar = "TALOSDIR"
```

Comment:
I think we can remove this.

Reply:
oh yeah, my bad, will remove in the next PR, thanks!